src/main/java/HookKiller/server/board/controller/ArticleController.java

```diff
@@ -0,0 +1,67 @@
+package HookKiller.server.board.controller;
+
+import HookKiller.server.board.dto.ArticleRequestDto;
+import HookKiller.server.board.dto.PostArticleRequestDto;
+import HookKiller.server.board.service.ArticleContentService;
+import HookKiller.server.board.service.ArticleService;
+import HookKiller.server.board.type.BoardType;
+import jakarta.servlet.http.HttpServletRequest;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Slf4j
+@RestController
+@RequestMapping("/articles")
+@RequiredArgsConstructor
+public class ArticleController {
+
+  private final ArticleService articleService;
+  private final ArticleContentService articleContentService;
+
+  /**
+   * 게시글 조회
+   */
+  @GetMapping("/{boardId}")
+  public List<ArticleRequestDto> getArticleList(@PathVariable Long boardId, HttpServletRequest request) {
+    BoardType language = BoardType.valueOf(request.getHeader("language"));
+    return articleService.getArticleList(boardId, language);
```

Comment:
이게 낫지 않을까요?, "language"는 따로 상수화 시키는게 좋을 것 같네요.
```suggestion
    return articleService.getArticleList(boardId, BoardType.valueOf(request.getHeader("language")));
```